CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## edge-19.5.2
+
+* CLI
+  * Fixed `linkerd check` and `linkerd dashboard` failing when any control plane
+    pod is not ready, even when multiple replicas exist (as in HA mode)
+* Controller
+  * Fixed control plane components failing on startup when the Kubernetes API
+    returns an `ErrGroupDiscoveryFailed`
+* Proxy
+  * Added a dispatch timeout that bounds the amount of time a request can be
+    buffered in the proxy
+  * Removed the limit on the number of concurrently active service discovery
```

Comment:
Should this be "concurrently active service discoveries"?

Reply:
Ah, this was supposed to say "concurrently active service discovery queries" but the word "queries" disappeared somehow. Fixed it in e0971d9d247bdc68cbc35693b98efb376c8b02b4